roles/confluent.control_center/tasks/health_check.yml

```diff
@@ -17,5 +17,29 @@
   until: result.status == 200
   retries: 60
   delay: 10
-  when:
-    - control_center_ssl_enabled|bool
+  when: control_center_ssl_enabled|bool and not rbac_enabled|bool
+
+- name: Wait for webpage to serve HTTP content - RBAC
```

Comment:
theres a variable for c3's http protocol, should be able to combine into one task

Reply:
resolving in this pr: https://github.com/confluentinc/cp-ansible/pull/247